modules/ticket_selector/assets/ticket_selector.js

```diff
@@ -128,4 +128,40 @@ jQuery(document).ready(function ($) {
 		$( this ).click();
 	} );
 
+
+	$('.display-tckt-slctr-tkt-details').on(
+		'click',
+		function () {
+			// get target element from "this" (the control element's) "rel" attribute
+			var target = $(this).attr("rel");
+			if (target.length) {
```

Comment:
```suggestion
			if (target?.length) {
```
maybe?

Reply:
don't think that will work because this code doesn't run through babel/webpack or whatever does all of that magic of converting ES6 to vanilla JS

Reply:
`target.length` is always defined in that case and defaults to `0`

Reply:
right.. jQuery always returns an object and checking its length is the easiest way to confirm that it is the expected DOM element